src/axolotl/integrations/base.py

```diff
@@ -338,6 +364,23 @@ def get_input_args(self):
                 input_args.append(input_args_from_plugin)
         return input_args
 
+    def load_datasets(self, cfg, preprocess: bool = False):
+        """
+        Calls the load_datasets method of each registered plugin.
+
+        Parameters:
+        cfg (dict): The configuration for the plugins.
+        preprocess (bool): Whether this is preprocess step of the datasets.
+
+        Returns:
+        dataset_meta (TrainDatasetMeta | None): The dataset metadata loaded from all registered plugins.
+        """
+        for plugin in self.plugins.values():
+            dataset_meta = plugin.load_datasets(cfg, preprocess)
+            if dataset_meta is not None:
+                return dataset_meta
```

Comment:
What if multiple plugins load datasets? Probably should assert here about uniqueness.